### PR TITLE
Expansion of the EC2 run readme, and addition of an example shell script

### PR DESCRIPTION
**Pull request by katebee**
### 👉 Adding more links to external resources

Saving everyone a few clicks

picture 137

### 👉 Reordering content and adding expanding some sections

I think this order makes it easier to follow along and learn as you go. I ran through the guide and had some questions, so adding some verbosity for any AWS/EC2 beginners.

### 👉 Changing the command used in the wait_for_command function

Script was not working as expected but was giving the helpful error:

```
Invalid choice: 'get-command-invocation', maybe you meant:

  * list-command-invocations
```

### 👉 Copying the shell script into an example file

The docs can link out to the example, and this will make it easier for others to copy the content

picture 138

### 👉 Adding line breaks before triple backticks

No visible change to web view. Some markdown viewers require this to format correctly.

## Will this require CloudFormation and/or updates to the AWS StackSet?

Nope

### CODE REVIEW

File: ssm-util.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
```
Comment (adamnfish) [inline ssm-util.sh]: I'd consider 'auto-formatting' this in IntelliJ or something along those lines. At least, the indentation is inconsistent in `wait_for_command` which might be worth a tweak.

Minor point though and no worries.

File: hq/markdown/run-command.md [hunk 2]
```diff
@@ -92,14 +122,14 @@ for this_instance in $instance; do
    while [[ $responseCode -eq -1 ]]; do
       printf "."
       sleep 1
-      responseCode=$(aws --region $region --profile $profile ssm get-command-invocation \
+      responseCode=$(aws --region $region --profile $profile ssm list-command-invocations \
```
Comment (adamnfish) [inline hq/markdown/run-command.md]: This is a significant change!

cc @aug24 for approval

Comment (shtukas) [reply hq/markdown/run-command.md]: Let's wait for @aug24 input, but the proposed version is correct. (What I think happened, but this is only a theory, is that @aug24 might have looked that the Windows PowerShell version, but @katebeeis correctly reporting the `aws` cli version.)

The AWS documentstion for `aws` says 

```
aws ssm list-command-invocations --command-id "command ID" --details
``` 

and @katebee proposes

```
aws ssm list-command-invocations \
    --region $region --profile $profile \
    --command-id "$command_id" \
    --instance-id "$this_instance" \
         | jq -r '.ResponseCode'
```

@adamnfish

**Comment by shtukas** [comment]
All good for me. I noticed that the `read_command_output` adopt the optimistic case that the command output will have less than 2500 lines, but as a proof of concept it's all right 

:+1:

**Pull request merged**